.dotnet/src/Generated/Models/ServiceLatencyTierOutcome.cs

```diff
@@ -0,0 +1,34 @@
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace OpenAI
+{
+    public readonly partial struct ServiceLatencyTierOutcome : IEquatable<ServiceLatencyTierOutcome>
```

Comment:
What do you think about these names:
ServiceLatencyTierChoice --> ServiceTier
ServiceLatencyTierOutcome --> EffectiveServiceTier